generators/server/templates/build.gradle.ejs

```diff
@@ -502,24 +502,24 @@ dependencies {
     }
     implementation "io.springfox:springfox-bean-validators"
     <%_ } else if (applicationType === 'gateway' || applicationType === 'monolith') { _%>
-    implementation "io.springfox:springfox-swagger2:3.0.0-SNAPSHOT"
-    implementation "io.springfox:springfox-swagger-ui:3.0.0-SNAPSHOT"
-    implementation "io.springfox:springfox-spring-webflux:3.0.0-SNAPSHOT"
+    implementation "io.springfox:springfox-swagger2:${springfox_version}"
+    implementation "io.springfox:springfox-swagger-ui:${springfox_version}"
+    implementation "io.springfox:springfox-spring-webflux:${springfox_version}"
     <%_ } _%>
     <%_ if (prodDatabaseType === 'mysql') { _%>
     implementation "mysql:mysql-connector-java"
         <%_ if (!reactive) { _%>
     liquibaseRuntime "mysql:mysql-connector-java"
         <%_ } else { _%>
-    implementation "dev.miku:r2dbc-mysql:0.8.1.RELEASE"
+    implementation "dev.miku:r2dbc-mysql:${r2dbc_mysql_version}"
```

Comment:
All r2dbc versions will be automatically managed by spring-boot-dependencies in SB 2.3

Reply:
FYI I've already started a spring-boot-2.3 branch where I remove these explicit versions.

Reply:
@murdos Wanted to start it also last week when I discover it... ;)